### PR TITLE
cloud-hypervisor: 0.8.0 -> 0.14.1

### PR DESCRIPTION
**Pull request by alyssais**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118497` at 4034d470 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/58abd7d19619e848ce69d5daf478071b">cloud-hypervisor</a></li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement.

File: pkgs/applications/virtualization/cloud-hypervisor/default.nix [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
 
   meta = with lib; {
     homepage = "https://github.com/cloud-hypervisor/cloud-hypervisor";
     description = "Open source Virtual Machine Monitor (VMM) that runs on top of KVM";
     changelog = "https://github.com/cloud-hypervisor/cloud-hypervisor/releases/tag/v${version}";
     license = with licenses; [ asl20 bsd3 ];
-    maintainers = with maintainers; [ offline ];
-    platforms = [ "x86_64-linux" ];
+    maintainers = with maintainers; [ offline qyliss ];
+    platforms = [ "aarch64-linux" "x86_64-linux" ];
```
Comment (SuperSandro2000) [inline pkgs/applications/virtualization/cloud-hypervisor/default.nix:25]: Maybe linux is just fine.

**Review by SuperSandro2000 — APPROVED** [review]

File: pkgs/applications/virtualization/cloud-hypervisor/default.nix (hunk 1 above)
Comment (alyssais) [reply pkgs/applications/virtualization/cloud-hypervisor/default.nix:25]: $5

Comment (SuperSandro2000) [reply pkgs/applications/virtualization/cloud-hypervisor/default.nix:25]: Thanks for clarifying.

**Pull request merged**